alacritty/src/display.rs

```diff
@@ -592,7 +592,7 @@ impl Display {
                 self.draw_search(config, &size_info, message_bar_lines, &search_text);
 
                 // Compute IME position.
-                Point::new(size_info.lines() - 1, Column(regex.len() - 1))
+                Point::new(size_info.lines() - 1, Column(search_text.len() - 1))
```

Comment:
Since we renamed `regex` to `search_text`, this broke. Unfortunately Rust's variable shadowing did not do us any favors here.